pom.xml

```diff
@@ -215,25 +215,74 @@
       <artifactId>script-security</artifactId>
       <version>1.54</version>
     </dependency>
-    <!-- not sure if this makes any difference but attempting to fix enforcer errors-->
-    <!-- TODO rather use dependencyManagement if necessary -->
-    <dependency>
-      <groupId>joda-time</groupId>
-      <artifactId>joda-time</artifactId>
-      <version>2.10.1</version>
-    </dependency>
-    <dependency>
-      <groupId>io.fabric8</groupId>
-      <artifactId>kubernetes-client</artifactId>
-      <version>4.1.3</version>
-    </dependency>
-    <dependency>
-      <groupId>com.fasterxml.jackson.core</groupId>
-      <artifactId>jackson-annotations</artifactId>
-      <version>2.9.0</version>
-    </dependency>
```

Comment:
These should actually have been in `dependencyManagement` if required, not top-level `dependencies`.